main.js

```diff
@@ -0,0 +1,436 @@
+
+let towerOne = [5, 4, 3, 2, 1];
+let towerTwo = [];
+let towerThree = [];
+
+let board = [towerOne, towerTwo, towerThree];
+
+document.addEventListener("DOMContentLoaded", function() {
+  const inputBoxDisc = document.getElementById("text-box-one");
+  const inputBoxTower = document.getElementById("text-box-two");
+
+  let discOneTowerLocation = 1;
+  let discTwoTowerLocation = 1;
+  let discThreeTowerLocation = 1;
+  let discFourTowerLocation = 1;
+  let discFiveTowerLocation = 1;
+
+  const moveDisc = function () {
+    let discOrigin = parseInt(inputBoxDisc.value);
+    let towerDestination = parseInt(inputBoxTower.value);
+
+    const allTheThings = function () {
+      const moveDisplay = function ()   {
+        console.log("Disc " + discOrigin + " moved to Tower " + towerDestination + ".");
+      };
+      moveDisplay();
+  
+      const gameDisplay = function () {
+        board.forEach((tower, index) => {
+          const label = `Tower ${index + 1}`;
+          if (tower.length === 0) {
+            console.log(`${label}:`);
+          } else {
+            console.log(`${label}: ${tower.join(", ")}`);
+          }
+        });
+      };
+      gameDisplay();
+
+    //   const updateBoard = function () {
+    //     board.map(row => {
+    //       if (row.length === 0) {
+    //         console.log("---");
+    //       } else {
+    //         console.log("---" + row.join(" ,"));
+    //       }
+    //     });
+    //   };
+    //   updateBoard();
+    };
+    
+    const clearTextBox = function () {
+      inputBoxDisc.value = "";
+      inputBoxTower.value = "";
+    };
+
+    if (isNaN(discOrigin) || isNaN(towerDestination)) {
+      alert("Error. You must enter a number.");
+      clearTextBox();
+      return;
+    } else if (discOrigin === 1 && towerDestination === discOneTowerLocation) {
+      alert("Error.  Disc 1 is already on that Tower.");
+      clearTextBox();
+      return;
+    } else if (discOrigin === 2 && towerDestination === discTwoTowerLocation) {
+      alert("Error. Disc 2 is already on that Tower.");
+      clearTextBox();
+      return;
+    } else if (discOrigin === 3 && towerDestination === discThreeTowerLocation) {
+      alert("Error.  Disc 3 is already on that Tower.");
+      clearTextBox();
+      return;
+    } else if (discOrigin === 4 && towerDestination === discFourTowerLocation) {
+      alert("Error.  Disc 4 is already on that Tower.");
+      clearTextBox();
+      return;
+    } else if (discOrigin === 5 && towerDestination === discFiveTowerLocation) {
+      alert("Error.  Disc 5 is already on that Tower.");
+      clearTextBox();
+      return;
+    } else if ((discOrigin === 2 && towerDestination === 1 && towerOne[towerOne.length -1] < discOrigin) || (discOrigin === 2 && towerDestination === 2 && towerTwo[towerTwo.length - 1] < discOrigin) || (discOrigin === 2 && towerDestination === 3 && towerThree[towerThree.length - 1] < discOrigin)) {
```

Comment:
The goal should be low complexity - there is also a lot of hard coding to values here, ideally you would want to be able to scale this which isn't sustainable with this solution